File: src/main/java/com/fish/service/other/AcceptedService.java
package com.fish.service.other;

import com.fish.entity.pojo.Accepted;
import com.mybatisflex.core.service.IService;

import java.util.ArrayList;

public interface AcceptedService extends IService<Accepted> {
    int addAccepted(Accepted accepted);
    ArrayList<Accepted> getAccepts();
    ArrayList<Accepted> getMyAccepts(String userId);
    ArrayList<Accepted> getTopicAccepts(Long topicId);
}

File: src/main/kotlin/com/fish/service/accepted/AcceptedService.kt
package com.fish.service.accepted

import com.fish.entity.pojo.Accepted
import com.mybatisflex.core.service.IService

interface AcceptedService : IService<Accepted> {
    fun addAccepted(accepted: Accepted): Int
    fun getAccepts(): ArrayList<Accepted>
    fun getMyAccepts(userId: String): ArrayList<Accepted>
    fun getTopicAccepts(topicId: Long): ArrayList<Accepted>
}
